app/src/main/res/layout/topic_fragment.xml

```diff
@@ -1,18 +1,48 @@
 <?xml version="1.0" encoding="utf-8"?>
-<layout
-  xmlns:android="http://schemas.android.com/apk/res/android"
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:app="http://schemas.android.com/apk/res-auto">
+
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
     android:layout_height="match_parent">
-    <TextView
-      android:id="@+id/dummy_text_view"
-      android:layout_width="wrap_content"
+
+    <com.google.android.material.appbar.AppBarLayout
+      android:id="@+id/appBarLayout"
+      android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:text="This is dummy TextView for testing"
+      android:theme="@style/ThemeOverlay.AppCompat.Dark.ActionBar"
```

Comment:
Why are we mixing the dark & light themes?

Reply:
Changed it to Dark in both places

Reply:
But aren't we using a light theme at the activity level? I'm trying to understand why we're using dark here but light at the activity level. We should be consistent everywhere, otherwise switching to dark mode might have weird side effects.

Reply:
Resolved